api/LargestContentfulPaint.json

```
@@ -187,6 +187,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cross-origin-render-time": {
+          "__compat": {
```

Two notes:
1. The `render-time` in the feature name seems redundant, given that the parent feature is `renderTime`, so `cross-origin` (or `cross-origin-media`?) would suffice.
2. Let's add a "description", otherwise it renders as `cross-origin[-render-time]` on MDN.

```suggestion
        "cross-origin": {
          "__compat": {
            "description": "Cross-origin",
```